api/meetings/meetingsModel.js

```diff
@@ -3,8 +3,17 @@ const db = require('../../data/db-config');
 const findAll = async () => {
   //returns names and profile info based on the host
   const meetings = await db('meetings as m')
-    .join('profiles as p', 'm.host_id', 'p.profile_id')
-    .select('m.*', 'p.first_name', ' p.last_name', 'p.email');
+    .join('profiles as mentor', 'm.host_id', 'mentor.profile_id')
+    .join('profiles as attendee', 'm.host_id', 'attendee.profile_id')
```

Comment:
let's take this one from` mentor_intake.mentee_intake_id` instead of `profiles.profile_id`

Reply:
I mean i would assume that in order to schedule a meeting you would have to be accepted into the program therefore, that information would be in profiles table. Cause I see it like mentee/mentor intake table is for the application proccess, and profile table is once that application is accepted, or not?

Reply:
Yes, I would assume the same thing (IMO) BUT this would mean that we might have to change the order of migrations which is more intensive... What do we want to do?